topic_modeling/scrape_training_data/scrape_articles.py

```diff
@@ -53,52 +53,72 @@ def test_log_error():
     assert log_error('This is a sentence') == print('This is a sentence')
 
 
-# need two different ways to set directory to pass
-# Travis CI (because .travis file is called from
-# different directory)
-current_file = os.getcwd().split('/')[-1]
-
-if current_file == 'scrape_training_data':
-    base_dir = os.getcwd()
-else:
-    base_dir = os.path.join(
+def scrape_articles():
+    """Scrapes text from the articles that correspond to the article
+    links generated by 'get_article_links.py'
+
+    Returns
+    -------
+    text files
+        For every article that has text (e.g., isn't simply just an
+        image) a new .txt file is created. All of the text from that
+        article is dumped into that text file. The text file is named
+        with the same naming convention The Dartmouth uses for their
+        articles URLs.
+
+    """
+
+    # need two different ways to set directory to pass
+    # Travis CI (because .travis file is called from
+    # different directory)
+    current_file = os.getcwd().split('/')[-1]
```

Comment:
there is `os.path.split` which is recommended to be used over string's split since it would then work also on exotic systems such as windows